src/test/java/programminglife/gui/GuiControllerTest.java

```diff
@@ -198,7 +198,10 @@ public void clickOnTest() {
         }
 
         sleep(5, TimeUnit.SECONDS);
+        //backspace to remove the initial values.
+        clickOn("#txtMaxDrawDepth").type(KeyCode.BACK_SPACE);
```

Comment:
1) I think this won't work, because the max drawdepth is 10 by default (which is 2 characters)
2) You should use a loop over the length of the value, something like
```
int foo = draw depth value;
while(foo > 0) {
  foo = foo / 10; // divide by ten to make it one digit shorter
  type(backspace); // remove one digit
}
```